pkg/minishift/openshift/oc.go

```diff
@@ -42,3 +46,23 @@ func AddSudoersRoleForUser(user string) error {
 	}
 	return nil
 }
+
+// Add Current Profile Context
+func AddContextForProfile(profile string, ip string, username string) error {
+	cmdName := instanceState.Config.OcPath
+	ip = strings.Replace(ip, ".", "-", -1)
```

Comment:
:+1:, right now the _--cluster_ parameter will target the right cluster nickname